### PR TITLE
[A11y]: Comply text colors w/ WCAG standards > 'text-muted'

### LINKED ISSUE #5538: [A11y]: 'text-muted' on a grey bg violates WCAG color contrast standards
### Audit type

ARC Toolkit

### User journey audit issue

#5500

### Problem description

`text-muted` text color on a grey background doesn't pass the ARC Toolkit test. Other pages using `text-muted` on a grey bg should be updated along with this.
image

### Expected behavior

The new [Figma DLS](https://www.figma.com/file/o1QRtdQI0ozKq0n7ATrKlx/Marketing-DLS?node-id=3212%3A21096) shows a slightly darker grey HEX for these items.

### Additional details

Our new [Brand Colors from Figma](https://www.figma.com/file/o1QRtdQI0ozKq0n7ATrKlx/Marketing-DLS?node-id=18%3A101) should be reflected on the [Handbooks Brand Guidelines](https://handbook.sourcegraph.com/departments/engineering/design/brand_guidelines/color/) page as well, to eliminate confusion.

### PR DESCRIPTION
**Pull request by st0nebraker**
Closes #5538: Fixes 'text-muted' wherever used on a grey background (this violates WCAG color contrast standards).
image

### Notes
- The new [Figma DLS](https://www.figma.com/file/o1QRtdQI0ozKq0n7ATrKlx/Marketing-DLS?node-id=3212%3A21096) shows a slightly darker grey HEX for most of these items anyway. So I brought the related colors in & updated the relevant `text-muted` instances.
- I reached out to some of our designers (Steven & Tracey), and asked if the [Handbooks Brand Color](https://handbook.sourcegraph.com/departments/engineering/design/brand_guidelines/color/) guidelines can be in sync with the new Figma DLS above to eliminate any confusion on our end.
- ~We may need another ticket to deal with DLS color updates.~ Ticket here #5559

### Changelog
- Updates `text-muted` instances to be in line with DLS & comply with WCAG color contrast standards.
- Reduces unneeded BEM css in files touched.

### Test
1. Ensure prettier has standardized the proposed changes.
2. Run an [ARC Toolkit test](https://docs.sourcegraph.com/dev/background-information/web/accessibility/how-to-arc-toolkit) to ensure changed colors don't violate requirement.
3. Ensure pages with removed BEM classes have a comparable appearance now.

### CODE REVIEW

File: src/pages/pricing.tsx [hunk 1]
```diff
@@ -123,36 +121,22 @@ const PricingPage: FunctionComponent = () => (
...
+                                logo={{src: '/external-logos/lyft-logo.svg', alt: 'Lyft logo: Read the case study', href: '/case-studies/criteo-tackles-big-code'}}
+                            />
                         </div>
                     </div>
 
                     <div className="row pt-4">
                         <div className="col-md-12 mx-auto mb-4">
-                            <div className="pricing-plan pricing-plan--team card">
-                                <h2 className="card-title mt-3 mb-3 pricing-plan__title">Team</h2>
+                            <div className="px-6 card">
```
Comment (zlonko) [inline src/pages/pricing.tsx:138]: It looks like the styling on "Team" section on `/pricing` is misaligned with the rest of the page. The text is smaller than comparable text on the page, and there appears to be some extra vertical padding on mobile. Perhaps we can match the style of the "Free" and "Enterprise" sections by using something like `text-xl` and `px-5 px-md-6`?

Screen Shot 2022-07-06 at 11 05 00 AM

**Review by zlonko — CHANGES_REQUESTED** [review]
The ARC Toolkit is super useful. It looks like the updated color checks out! I made a couple of small notes on the Pricing page and noticed a few alterations suggested by Prettier.

File: src/pages/pricing.tsx (hunk 1 above)
Comment (st0nebraker) [reply src/pages/pricing.tsx:138]: Good eye! I anticipate this page will undergo a refresh/standardization soon so I don't want to spend too much time on it. But I did align these sections to have more similar spacing in [758e671](https://github.com/sourcegraph/about/pull/5556/commits/758e671aec8c5e8eadc8ffea4ce898d5659e986a) commit. Take another look! The tags are nested a bit weird, so I hope to clean up the template more later.

File: src/pages/accelerate-dev-onboarding.tsx [hunk 2]
```diff
@@ -51,36 +51,24 @@ const AccelerateDevOnboarding: FunctionComponent = () => {
...
+                <div className="container py-7 max-w-650">
+                    <BlockquoteWithLogoBottom
+                        header="Learn how Convoy increases the efficiency and confidence of entry level developers"
+                        quote="For our new developers, Sourcegraph has been invaluable to get to know the repository
+                        structure, to track down where code lives, and self-service during their
+                        investigations."
+                        author="Owen Kim, Senior Software Engineer at Convoy"
+                        logo={{src: '/external-logos/convoy-logo.svg', alt: 'Convoy logo'}}
+                        link={{text: 'Read the full case study', href: '/case-studies/convoy-improved-on-boarding'}}
+                    />
```
[the remark this answers is not in the document]
Comment (st0nebraker) [reply src/pages/accelerate-dev-onboarding.tsx:63]: Thanks! :)

File: src/components/Pricing/PricingPlanFeature.tsx [hunk 3]
```diff
@@ -42,7 +34,7 @@ export const PricingPlanFeature: FunctionComponent<Props> = ({
                     overlay={<Tooltip id="tooltip">{description}</Tooltip>}
                 >
                     {({ ref, ...triggerHandler }) => (
-                        <span {...triggerHandler} ref={ref} className="ml-2 pricing-plan-feature__help">
+                        <span {...triggerHandler} ref={ref} className="ml-2 text-gray-3">
```
[the remark this answers is not in the document]
Comment (st0nebraker) [reply src/components/Pricing/PricingPlanFeature.tsx]: Hm, I'm not sure if design has another opinion. But I updated this to `text-gray-4` for now! It does seem a bit dim to me too.

File: src/styles/globals.scss [hunk 4]
```diff
@@ -203,7 +203,6 @@ table tbody + tbody {
 @import 'components/blog/PodcastPost';
 @import 'components/blog/PostsList';
 @import 'components/blog/ReleasePost';
-@import 'components/pricing/index';
```
[the remark this answers is not in the document]
Comment (st0nebraker) [reply src/styles/globals.scss:206]: 😸 Every time I delete heaps of code I think of Marie Kondo. Tossing the piles that don't spark joy 🗑️ 

image

**Review by zlonko — DISMISSED** [review]

**Review by bretthayes — CHANGES_REQUESTED** [review]
Love seeing the removal of those bem-styled styles and cleanup! Bless! 🙏🏻
Can't wait to tackle an audit for our colour variables soon too!
Looks great so far! Just a few small things I noticed:

File: src/pages/fixing-vulnerabilities.tsx [hunk 5]
```diff
@@ -51,49 +51,35 @@ const FixingVulnerabilities: FunctionComponent = () => {
...
-                        <FormLegal />
-                    </div>
+                <div className="mt-5 max-w-400 mx-auto">
+                    <Link href="/demo">
+                        {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
+                        <a
+                            className="btn btn-primary mt-5 d-block d-sm-inline-block"
+                            data-button-style={buttonStyle.primary}
+                            data-button-location={buttonLocation.bodyDemo}
+                        >
```
[the remark this answers is not in the document]
Comment (st0nebraker) [reply src/pages/fixing-vulnerabilities.tsx:81]: Good eye - I'll cover the other missing ones in #5487

**Review by bretthayes — APPROVED** [review]

**Pull request merged**